lib/recipes.js

```diff
@@ -1,3 +1,34 @@
+class RecipeStep {
```

Comment:
unfortunatelly I don't get this lib.js at all :/ I hope it works but I just don't understand the code

Reply:
First I defined a class for recipe steps and a class for a recipe. A recipe can consist of several recipe steps. I created a local example data and I then put these example recipes into a list. And yes it works absolutely fine :)